### ISSUE TITLE
OrangeHRM Arabic Localisation is bugged

### ISSUE THREAD

**Issue by y0gng** [orangehrm/orangehrm#1720]
### OrangeHRM version

5.5

### Environment details

- OrangeHRM source:
- Platform:
- PHP version:
- Database and version:
- Web server:
- Browser:

### Describe the bug

So I am trying to translate OrangeHRM to Arabic and I followed the method that was put by OrangeHRM and when I partially translated it and wanted to try it out I put the localization to Arabic and it started giving out some errors in buttons and and translated texts such as shown below:
"auth.login, auth.username.."
![image](https://github.com/orangehrm/orangehrm/assets/46849007/b29ea43c-df26-4e1b-af35-f1586b918d09)
![image](https://github.com/orangehrm/orangehrm/assets/46849007/ea6d849e-8c22-451e-8e58-ef68f712c94b)

**Comment by kavithagurusamy** [comment]
Please share your environment details so that we could identify the issue you are facing.

**Comment by kavithagurusamy** [comment]
Can you please share us the requested information?

**Comment by enesphp** [comment]
i have the same problem  https://github.com/orangehrm/orangehrm/issues/1737

**Comment by enesphp** [comment]
I have solved the issue. The root cause of the problem occurs when we copy the translated text from the translation screen and paste it into the input field, then click "save."

Here's what I have identified:

I queried the database with the corresponding Turkish Language ID in the table where translations are stored. "SELECT * FROM ohrm_i18n_translate WHERE language_id = 480;" Turkish Language ID number is 480.

Then, I exported all the translations I made for the Turkish language in SQL format, opened it in a text editor. I noticed that some data contains hidden invalid characters like <80a00>. My guess is that these characters might have come from copy-pasting during translation. When we save the translated text with unnecessary characters like <80a00> into the input field, it gets stored in the database and caches are updated.

As a solution, I applied the following code in the database: "DELETE FROM ohrm_i18n_translate WHERE language_id=480;" With this code, I deleted all the Turkish content for which I had done translations.

I reviewed the downloaded SQL file, removed the lines containing faulty data, re-imported the SQL, and deleted the src/cache/orangehrm folder via FTP.

The system is now functioning properly.
